js/apps/account-ui/test/utils.ts

```diff
@@ -1,13 +1,18 @@
 import { getRootPath } from "../src/utils/getRootPath";
 
-export function getBaseUrl(): string {
+export function getTestServerUrl(): string {
   return process.env.KEYCLOAK_SERVER ?? "http://localhost:8080";
 }
 
+export function getKeycloakServerUrl(): string {
```

Comment:
Let's keep this under the name `getBaseUrl()`. I think that is explanatory enough.

Reply:
"Base" can refer either to account console server or keycloak server, or the base URL of the two if console and REST endpoints are served by the same server. I'd rather keep this name unambiguous.

Reply:
How about `getOrigin()`? That is a [well defined term](https://developer.mozilla.org/en-US/docs/Web/API/URL/origin) .

Reply:
Not sure.. Origin does not specify if it is the REST endpoint origin or account console origin.